app/controllers/admin/chapters_controller.rb

```diff
@@ -59,13 +66,29 @@ def members
     render plain: @emails
   end
 
+  def update_active
+    authorize @chapter
+
+    @chapter.update(update_active_params)
+
+    head :ok
+  end
```

Comment:
Can we stick to standard rails CRUD operations and reuse `update` to achieve this instead. The `active` param can just be part of chapter_params and the existing `update` method could be used to achieve the same thing.